wordpress-mobile/WordPress-iOS
# Giphy: Add Giphy import to Aztec

Implements #10090.

This PR adds Giphy import to Aztec. It also pulls in changes to Aztec which show a GIF badge on imported GIFs. **Before this PR is merged**, Aztec should be updated and the Podfile of this branch should be updated.

![giphy-aztec](https://user-images.githubusercontent.com/4780/47119845-6ce04600-d264-11e8-9b00-fe4eb97a2d89.gif)

**To test:**

* Build and run
* In the editor, tap + in the formatting toolbar, choose `...` and then `Giphy`.
* Attempt to insert some gifs and ensure they're inserted correctly. Check you see the GIF badge on the gifs, and that they animate correctly when published.
